lib/braze_ruby/rest/canvas_details.rb

```diff
@@ -6,14 +6,12 @@ class CanvasDetails < Base
       attr_reader :api_key, :params
```

Comment:
I guess you could also remove attr reader for `api_key` in all those places

Reply:
Removed the redundant `attr_reader` and also removed  `api_key` being passed on some other http request call body/params.